libs/pages/elewa/social-impact/src/lib/components/social-impact-page-top-banner/social-impact-page-top-banner.component.html

```diff
@@ -0,0 +1,12 @@
+<div id="elewa-hero">
+  <img class="backgroundImage" [src]="backgroundImg" alt="top banner hero image" />
+
+  <div class="container">
+    <div>
+      <div class="content">
+        <h2 class="subtitle">{{ subtitle }}</h2>
+        <h1 class="title">{{ title }}</h1>
+      </div>
```

Comment:
For this issue you're reusing a component already created in #10. This does not follow requirements.